src/components/BorrowerProfile/LendCta.vue

```diff
@@ -793,6 +804,17 @@ export default {
 		},
 		isLendAmountButton() {
 			return (this.lendButtonVisibility || this.state === 'lent-to') && (isLessThan25(this.unreservedAmount)); // eslint-disable-line max-len
+		},
+		bookmarkVersion() {
+			// Display follow for all US loans no matter login state
+			if (this.distributionModel === 'direct') {
```

Comment:
Doesn't necessarily need to happen now, but since these magic strings are being used in more than one file, might be worth creating a "fake" enum file for them like with `LoanStatusEnum.js`